### PR TITLE
feat: add IP Address Field

### PR DESCRIPTION
**Pull request by aokblast**
This patch add IPAddress Field on Submission Pages as BackEnd have this change.

### CODE REVIEW

**Comment by uier** [comment]
Hi @aokblast, the changes look good to me, only one question, does the IP Address need to be displayed for every users?

Or it's mainly for TA & instructor? In this case, it would be better to only show the IP Address for admin users by adding `v-if="session.isAdmin"` on `<td>` and `<th>`. Because the table has too many columns and seems clutter.

**Comment by uier** [comment]
oh, another question, you may also wanna add this ip column in `src/pages/course/[name]/submissions/[id].vue`? This page component is used for page displaying single submission details.

**Review by uier — CHANGES_REQUESTED** [review]

File: src/pages/course/[name]/submission/[id].vue [hunk 1]
```diff
@@ -130,14 +132,15 @@ async function rejudge() {
...
-            <table v-else class="table-compact table w-full" v-for="(task, taskIndex) in submission.tasks">
+            <table v-else class="table table-compact w-full" v-for="(task, taskIndex) in submission.tasks">
               <thead>
                 <tr>
                   <th>{{ $t("course.submission.detail.id") }} {{ taskIndex }}</th>
                   <th>{{ $t("course.submission.detail.status") }}</th>
                   <th>{{ $t("course.submission.detail.runtime") }}</th>
                   <th>{{ $t("course.submission.detail.memory") }}</th>
                   <th>{{ $t("course.submission.detail.score") }}</th>
+                  <th>{{ $t("course.submission.detail.ipAddr") }}</th>
```
[the remark this answers is not in the document]
Comment (aokblast) [reply src/pages/course/[name]/submission/[id].vue]: Fix it

**Review by uier — APPROVED** [review]
Thanks!

**Pull request merged**